checker/jtreg/nullness/AssignmentPerformanceTest.java

```diff
@@ -1,6 +1,9 @@
-// @skip-test
-// Test case for issue #536: https://github.com/typetools/checker-framework/issues/536
-
+/*
+ * @test
+ * @summary Test case for issue #536: https://github.com/typetools/checker-framework/issues/536
+ *
+ * @compile/timeout=60 -J-Xmx256M -XDrawDiagnostics -Xlint:unchecked -processor org.checkerframework.checker.nullness.NullnessChecker -AprintErrorStack -Alint AssignmentPerformanceTest.java
```

Comment:
I think this should be needed to be a jtreg test to confirm it'll be done until the timeout.
Please note that this test case is already moved to jtreg directory.